build-system/compile/generate/bento.js

```diff
@@ -55,6 +51,35 @@ function generateBentoRuntimeEntrypoint(
   `);
 }
 
+/**
+ * @param {Object<string, string[]>} [packageSymbols]
```

Comment:
Yes, this is one of like 4 possible ways to denote an optional arg 😆 Separate from this PR it may be worth investigating if typescript-eslint supports enforcing just one